pkg/fw/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb as build
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+FROM lfedge/eve-alpine:c61659241a11346dfb5a595d1217d64381df3adb as build
```

Comment:
Well, seems we give up on using the same hash for `eve-alpine` everywhere anyways:
```
> find pkg/ -name Dockerfile | xargs perl -ne 'print $1."\n" if (m/^FROM lfedge\/eve-alpine:(\S+)/);' | sort | uniq -c
      1 0f4e313d0d84ac313ea35e966def9ef96f61aafb
      1 12487b9900ba40f3ecdadfec2c84799fa34e5014
      2 145f062a40639b6c65efa36bed1c5614b873be52
      1 29dc7661e9c3dcd0ec7ef9b98d2df58354d85fc0
      1 862d6db0e2e938d11ef83e83fe9c7a8c83631517
     30 9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb
      2 c114cf1d3ea51534f061f9aa949beb6ac5c12fb3
      1 d32cf7889da970a42fdf5c1c5454a311356cb8f8
      1 f49b278bcb18b5f17a997c7b61df95bb53b2fa31
      2 fad44e3702708a8d044663a20fd98d933dddb41e
```

Reply:
No, you know, that's painful to update all at once... but I commented at some recent PR that we should align those hashes again.... but definitely needs to be done in a dedicated PR, actually broke into a couple of them...

Reply:
There should be a way to enforce same version for every Dockerfile. From the top of my head we can use .env file and run docker like 
`env $(cat .env | xargs) docker // bla bla bla`

It would be still painful to go through all make files and we will need to document somewhere in case someone introduces new docker file. 

Another option would be reusing locally used container like build-containers.

Reply:
> No, you know, that's painful to update all at once... but I commented at some recent PR that we should align those hashes again.... but definitely needs to be done in a dedicated PR, actually broke into a couple of them...

I agree; it's fine for me to keep it like this here, just wanted to make people aware.